server/dataone_package.py

```diff
@@ -93,63 +114,179 @@ def create_minimum_eml(tale, user, item_ids, eml_pid):
     ET.SubElement(ind_name, 'givenName').text = firstName
     ET.SubElement(ind_name, 'surName').text = lastName
 
-    # Add a <otherEntity> block for each object
-    for item_id in item_ids:
-        item = Item().load(item_id, user=user)
-        description = item['description']
-
+    def create_other_entity(name, description):
```

Comment:
For common types, like csv, tsv, xls, xlsx, we could assume the Entity is a `dataTable` (tabular) instead of an `otherEntity` (unknown).

Reply:
Would you recommend creating a method along the lines of ...

```
def is_tabular_format (object_format):
   tabular_formats = [list of extensions]
   if object_format is any(tabular_formats):
      return true
``` 

And then create an `otherEntity` or `dataTable` depending on the result

Reply:
Basically. In your example above, I'd break the `[list of extensions]` `List`  out of the method scope and into module scope so it's not re-defined repeatedly. Your example looks good to me.

Another pattern you could use is:

```python
def create_entity(name, description, entity_type = None):
    if entity_type is None:
        entity_type = detect_entity_type(name) # Fallback value of `otherEntity`

    entity = ET.SubElement(dataset, entity_type)
    ...elided
```

where `detect_entity_type` is like `is_tabular_format` but actually just returns the element name to use instead of a `bool`.